src/components/KCheckbox/KCheckbox.vue

```diff
@@ -10,7 +10,18 @@
       class="k-input"
       @change="handleChange"
     >
-    <slot />
+    <span
+      v-if="(label || $slots.label)"
+      class="k-checkbox-label"
+    >
+      <slot name="label">{{ label }}</slot>
+    </span>
+    <div
+      v-if="(label || $slots.label) && (description || $slots.description)"
+      class="k-checkbox-description"
+    >
+      <slot name="description">{{ description }}</slot>
+    </div>
```

Comment:
This change removes the default `<slot />` and looks like a breaking change to me. If we have something like `<KCheckbox>someLabel</KCheckbox>`, after this PR the text `someLabel` will not be rendered.

The `KRadio` below keeps the default `<slot />` intact. Could we do that here as well?

Reply:
Agree, but I'll add that I believe the `label` slot (in both components) should actually just be the `default` slot (to keep this non-breaking) and then have the addition of the `description` slot.

Reply:
Makes sense, updated to have `label` as the **default** slot and `description` as a **named** slot.